dependencies/precise/hammer_cli_foreman/copyright

```diff
@@ -1,33 +1,25 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: hammer_cli_foreman
-Source: FIXME <http://example.com/>
+Source: https://github.com/theforeman/hammer-cli-foreman
 
 Files: *
-Copyright: <years> <put author's name and email here>
-           <years> <likewise for another author>
-License: GPL-2+ (FIXME)
- This program is free software; you can redistribute it
- and/or modify it under the terms of the GNU General Public
- License as published by the Free Software Foundation; either
- version 2 of the License, or (at your option) any later
- version.
+Copyright: 2012 Brian Gupta <brian.gupta@brandorr.com>
+           2013-2014 RedHat Inc. <foreman-dev@googlegroups.com>
```

Comment:
I'd maybe say "Foreman developers" here, as it isn't exclusively Red Hat (extra space)